roles/ovirt-engine-setup/README.md

```diff
@@ -0,0 +1,110 @@
+| ovirt_engine_dwh                | UNDEF                 | Bool value for installing local DWH. |
+| ovirt_engine_answer_file_path   | UNDEF                 | /path/to/custom/answerfile. |
+| ovirt_engine_db_host            | localhost             | IP or hostname of PostgreSQL server for engine database. |
+| ovirt_engine_db_port            | 5432                  |  Server listening port. |
+| ovirt_engine_db_name            | engine                | DB name for ovirt-engine. |
+| ovirt_engine_db_user            | engine                |  DB user which can access ovirt-engine DB. |
+| ovirt_engine_db_password        | UNDEF                 | password for user of ovirt-engine DB. |
+| ovirt_engine_dwh_db_host        | localhost             | IP or hostname of PostgreSQL server for DWH database. |
+| ovirt_engine_dwh_db_port        | 5432                  | Server listening port. |
+| ovirt_engine_dwh_db_name        | ovirt_engine_history  |  DB name for ovirt-engine-dwh. |
+| ovirt_engine_dwh_db_user        | ovirt_engine_history  |  DB user which can access ovirt-engine-dwh DB. |
+| ovirt_engine_dwh_db_password    | UNDEF                 | password for user of ovirt-engine DB. |
+
+* ISO domain related options:
+
+| Name                              | Default value         |  Description                               |
+|-----------------------------------|-----------------------|--------------------------------------------|
+| ovirt_engine_configure_iso_domain | False                 | Whether to confiure ISO domain on engine.   |
+| ovirt_engine_iso_domain_path      | /var/lib/exports/iso  | Create local ISO domain on engine machine.  |
+| ovirt_engine_iso_domain_name      | IDO_DOMAIN            | Name of ISO domain.                         |
+| ovirt_engine_iso_domain_acl       | 0.0.0.0/0.0.0.0(rw)   | ACL permissions for ISO domain mount point. |
+
+* Configure firewall
+
+| Name                             | Default value  |  Description |
+|----------------------------------|----------------|---------------------------------------------------------------------------------------------------------------------|
+|  ovirt_engine_firewall_manager   | firewalld      | In case you want to configure firewall, select the firewall manager 'firewalld', 'iptables', or put null otherwise. |
+
+* Update
+
+| Name                  | Default value         |  Description                                              |
+|-----------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_update   | False                 | True if you want to update setup packages and run engine-setup also when engine is already installed and running. |
+
+
+Dependencies
+------------
+
+* ovirt-engine-install-packages
+
+Example Playbook
+----------------
+
+```yaml
+---
+- hosts: engine
+  vars:
+    ovirt_engine_type: 'ovirt-engine'
+    ovirt_engine_version: '4.1'
```

Comment:
I just wonder why installer need to have version variable, any reason to not autodetect it?

Reply:
it is just to load the answer file. 
I would like to assume that whoever runs this actually knows what he is doing and what version is he installing.

Reply:
Since we decided to install engine packages in this role as well, we don't really have way to know engine version.
So, I guess this variable will be needed.